src/ask.rs

```diff
@@ -0,0 +1,17 @@
+pub async fn respond_ask() -> anyhow::Result<String> {
+    Ok("Don't ask to ask, just ask! \nhttps://dontasktoask.com/".to_string())
+}
+
+#[cfg(test)]
```

Comment:
This test can be dropped, as it doesn't help test anything meaningful. I think given the very small scope of the PR, it's okay without new tests